packages/extension-browser/src/devtools/views/controls/feedback.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import { getMessage } from '../../utils/i18n';
+
+import ExternalLink from './external-link';
+
+import * as styles from './feedback.css';
+
+const openIssueUrl = 'https://github.com/webhintio/hint/issues/new?labels=type%3Abug&amp;template=2-bug-report-browser.md&amp;title=%5BBug%5D+Bug+description';
+
+/**
+ * Link to github to give feedback.
+ */
+const Feedback = () => {
+    return (
+        <span className={styles.root}>
+            <ExternalLink href={openIssueUrl}>
```

Comment:
Nit: You can drop the wrapping span and put the class directly on `ExternalLink` if you want.
```suggestion
            <ExternalLink className={styles.root} href={openIssueUrl}>
```

That said, I've been trying to follow the best practice where parent components control layout, not children. In that case I'd actually prefer if this component accepted a className and the position were defined as part of the parent component's stylesheet (even if that proves to be slightly redundant at the moment).